scripts/lib/labels.ts

```diff
@@ -10,75 +10,31 @@ interface Label {
 	changelogTitle: string
 }
 
-const SCOPE_LABELS = [
-	{
-		name: `sdk`,
-		description: `Changes the tldraw SDK`,
-		changelogTitle: '📚 SDK Changes',
-	},
-	{
-		name: `dotcom`,
-		description: `Changes the tldraw.com web app`,
-		changelogTitle: '🖥️ tldraw.com Changes',
-	},
-	{
-		name: `docs`,
-		description: `Changes to the documentation, examples, or templates.`,
-		changelogTitle: '📖 Documentation changes',
-	},
-	{
-		name: `vs code`,
-		description: `Changes to the vscode plugin`,
-		changelogTitle: '👩‍💻 VS Code Plugin Changes',
-	},
-	{
-		name: `internal`,
-		description: `Does not affect user-facing stuff`,
-		changelogTitle: '🕵️‍♀️ Internal Changes',
-	},
-] as const satisfies Label[]
-
 const TYPE_LABELS = [
-	{ name: `bugfix`, description: `Bug fix`, changelogTitle: '🐛 Bug Fixes' },
-	{ name: `feature`, description: `New feature`, changelogTitle: '🚀 Features' },
 	{
 		name: `improvement`,
-		description: `Improving existing features`,
-		changelogTitle: '💄 Improvements',
-	},
-	{
-		name: `chore`,
-		description: `Updating dependencies, other boring stuff`,
-		changelogTitle: '🧹 Chores',
+		description: `Product Improvement`,
+		changelogTitle: '💄 Product Improvements',
 	},
+	{ name: `bugfix`, description: `Bug fix`, changelogTitle: '🐛 Bug Fixes' },
 	{
-		name: `galaxy brain`,
-		description: `Architectural changes`,
-		changelogTitle: '🤯 Architectural changes',
-	},
-	{ name: `tests`, description: `Changes to any test code`, changelogTitle: '🧪 Tests' },
-	{
-		name: `tools`,
-		description: `Changes to infrastructure, CI, internal scripts, debugging tools, etc.`,
-		changelogTitle: '🛠️ Tools',
+		name: `api`,
+		description: `API Change`,
+		changelogTitle: '🛠️ API Changes',
 	},
-	{ name: `dunno`, description: `I don't know`, changelogTitle: '🤷 Dunno' },
+	{ name: `other`, description: `Internal or flagship .com change`, changelogTitle: '🤷 Other' },
```

Comment:
is there a way to have these excluded from changelogs / release notes?

Reply:
Ah yeah, I forgot - lemme add that.